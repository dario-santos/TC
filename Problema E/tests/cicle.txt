2
S -> A
A -> S
