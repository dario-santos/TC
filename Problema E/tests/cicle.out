NULL(S) = True
NULL(A) = True
FIRST(S) = a
FIRST(A) = a
FOLLOW(S) = #
FOLLOW(A) = #
